[CQA] Fix feature-centric abstracts in Provisioning Hosts (#4958)
theforeman/foreman-documentation: guides/common/modules/ref_supported-client-platforms-in-provisioning.adoc

= Supported client platforms in provisioning

[role="_abstract"]

> You can provision hosts with the following operating systems and architectures.

Reviewer: ```suggestion
You can provision supported operating systems on hosts with supported architectures.
```
